src/sequences/SequenceEntity.js

```diff
@@ -14,6 +14,23 @@ export const sequencedActivityClasses = {
 };
 
 export class SequenceEntity extends Entity {
+	/**
+	 * @return An object with indices: completed, total, optionalViewed, optionalTotal, isCompleted
+	 */
+	completion() {
+		const subEntities = this._entity && this._entity.getSubEntitiesByRel && this._entity.getSubEntitiesByRel(Rels.completion);
+		const subEntity = subEntities.pop();
```

Comment:
is `subEntities` is referencing object that could be modified ( assume pop will remove item from it)

Reply:
Good point. I guess hoping it was a copy so it didn't matter. But lets be safe.